python_modules/dagster/dagster/cli/schedule.py

```diff
@@ -390,7 +390,12 @@ def execute_wipe_command(cli_args, print_fn):
         print_fn("Scheduler not defined for repository {name}".format(name=repository.name))
         return
 
-    scheduler = schedule_handle.get_scheduler()
-    scheduler.wipe()
-
-    print_fn("Wiped all schedules")
+    confirmation = click.prompt(
+        'Are you sure you want to delete all schedules and crontabs? Type DELETE'
+    )
+    if confirmation == 'DELETE':
+        scheduler = schedule_handle.get_scheduler()
+        scheduler.wipe()
+        print_fn("Wiped all schedules")
```

Comment:
use `click.echo`

Reply:
`print_fn` == `click.echo` here

Reply:
hm, why just here?

Reply:
I used `print_fn` because this is what was used in other functions in this file